mappings/net/minecraft/item/Item.mapping

```diff
@@ -149,6 +149,8 @@ CLASS net/minecraft/class_124 net/minecraft/item/Item
 	METHOD method_447 getAttackDamage (Lnet/minecraft/class_57;)I
 		ARG 1 attackedEntity
 	METHOD method_448 isHandheld ()Z
+	METHOD method_449 isHandheldFishingRod ()Z
```

Comment:
`isHandheldRod` might be enough, considering in modern, tools use the `item/handheld` model while in beta they override the `isHandheld` method, meanwhille rods used the `item/handheld_rod` model while in beta they would override this method, so `isHandheldRod` seems fitting to me as it also kinda disconnects it a bit from the fishing rod name-wise indicating that its fit for more general use by modders.